doc/requirements.rst

```diff
@@ -78,3 +76,22 @@ NVIDIA unlike AMD do not seem to suffer from the same stability issues as AMD
 GPUs when operating as a passthrough GPU, however due to the closed source
 nature of their drivers NVIDIA can not make use of the DMABUF feature in the
 Linux kernel unless you use the open source NVIDIA drivers.
+
+.. _igpu_kvmfr_recommended:
+
+iGPUs should use DMABUF
+^^^^^^^^^^^^^^^^^^^^^^^
+
+While `DMABUF` with the :doc:`KVMFR module <ivshmem_kvmfr>` offers performance
+benefits for all users, for the often bandwidth-starved users with an iGPU on
+their host it's considered a requirement for a decent experience.
+
+When using a normal SHM file, many GPU drivers will copy incoming frames from
+shared memory to an intermediary buffer, then upload it from that buffer to the
+GPU's framebuffer. The KVMFR module will instead use Direct Memory Access (DMA)
+to download incoming frames directly from shared memory, which may depending on
+GPU design eliminate the intermediary buffer. This is especially helpful to iGPU
```

Comment:
> which may depending on
GPU design eliminate the intermediary buffer.

I hope this modification is correct, this is how I understand it.